atlasdb-dbkvs-tests/src/test/resources/docker-startup/01_create_user.sql

```diff
@@ -0,0 +1,15 @@
+CREATE USER palantir IDENTIFIED BY "7_SeeingStones_7";
+
+GRANT
+    CREATE SESSION,
+    CREATE PROCEDURE,
+    CREATE SEQUENCE,
+    CREATE TABLE,
+    CREATE VIEW,
+    CREATE TYPE
+TO palantir;
+
+ALTER USER palantir
+    QUOTA UNLIMITED ON users;
+
+EXIT;
```

Comment:
minor nit, I think `EXIT;` isn't necessary as the image will add an implicit exit at the end of each .sql script